torch/storage.py

```diff
@@ -273,9 +273,9 @@ def _to(self, dtype):
             storage = storage.clone()
         return storage
 
-    def to(
-        self, *, device: torch.device, non_blocking: _bool = False
-    ) -> Union[_StorageBase, TypedStorage]:
+    def to(self, *, device: Union[str, torch.device], non_blocking: _bool = False):
+        if isinstance(device, str):
+            device = torch.device(device)
```

Comment:
```suggestion
    def to(self, *, device: torch._prims_common.DeviceLikeType, non_blocking: _bool = False):
        if not isinstance(device, torch.device):
            device = torch.device(device)
```
Is what it should be